k8s/aat/common/family-public-law/fpl-case-service.yaml

```diff
@@ -34,6 +34,7 @@ spec:
         DOCMOSIS_TORNADO_URL: https://docmosis-development.platform.hmcts.net
         IDAM_CLIENT_ID: fpl_case_service
         IDAM_CLIENT_REDIRECT_URI: https://fpl-case-service-*.service.core-compute-aat.internal/oauth2/callback
+        GATEWAY_URL: http://gateway-ccd.aat.platform.hmcts.net
```

Comment:
Please do same for prod otherwise POD won't start in PROD.

Reply:
And it should be `https://gateway-ccd.aat.platform.hmcts.net`

Reply:
`https` for both aat and prod?

Reply:
yes, always favour https over http unless https is not supported

Reply:
updated